include/seahorn/Bmc.hh

```diff
@@ -14,20 +21,66 @@
 
 namespace seahorn {
 using namespace expr;
+using ZModel_ref = std::shared_ptr<ZModel<EZ3>>;
```

Comment:
I think our naming convention is not to mix camel and snake case. That is, either caps or underscore, but not both.